src/containers/login/index.tsx

```diff
@@ -2,14 +2,13 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import LoginForm, { LoginFormData } from '../../components/login';
 import { SupabaseClient } from '@supabase/supabase-js';
-import { Database } from '../../database.types';
-
 
 interface LoginContainerProps {
-  supabase: SupabaseClient<Database>;
```

Comment:
We aren't fetching any data from the client here – we are just using auth – so we don't need to care about the types.